packages/api-core/src/api.js

```diff
@@ -292,7 +292,7 @@ export default class AvApi {
               (pageResp) => pageResp.data[key] || []
             )
           )
-        ).then((pages) => [...result, ...pages]);
+        ).then((pages) => [...result, ...pages].flat());
```

Comment:
`someArray.concat(...anotherArray)` returns a new `thirdArray` where all elements were merged together, so `[...result, ...pages]` should have been equivalent.. is the `pages` variable not always an array?

Reply:
pages is an array of additional pages, so each `page` is also an array

Reply:
Oh based on your `flat()` change I see. 

```js
const array1 = ['a', 'b', 'c'];
const array2 = ['d', 'e', 'f', ['g']];
const array3 = array1.concat(array2);
const array4 = array1.concat(...array2);

console.log(array3);
console.log([...array1, ...array2]);
console.log(array4);
```
produces
```
> Array ["a", "b", "c", "d", "e", "f", Array ["g"]]
> Array ["a", "b", "c", "d", "e", "f", Array ["g"]]
> Array ["a", "b", "c", "d", "e", "f", "g"]
```
so I guess `pages` had some nested data. Looks good!